g_docs.go

```diff
@@ -680,16 +701,36 @@ func getModel(str string) (pkgpath, objectname string, m swagger.Schema, realTyp
 		ColorLog("[ERRO] the model %s parser.ParseDir error\n", str)
 		os.Exit(1)
 	}
+
+	var packageName string
 	m.Type = "object"
 	for _, pkg := range astPkgs {
 		for _, fl := range pkg.Files {
-			for k, d := range fl.Scope.Objects {
-				if d.Kind == ast.Typ {
-					if k != objectname {
-						continue
-					}
-					parseObject(d, k, &m, &realTypes, astPkgs)
+			for _, d := range fl.Scope.Objects {
+				if d.Kind != ast.Typ || d.Name != objectname {
+					continue
 				}
+
+				pathInfo, err := generatePathInfo(fl)
+				if err != nil {
+					ColorLog("[ERRO] failed when generating path info: %v", err)
+					os.Exit(1)
+				}
+
+				pathInfo[pkg.Name] = pkgpath
+
+				res := new(objectResource)
```

Comment:
Whats that? I dont see `rest` is used beside here? What does it do and why it's important to parse and throw away?

*Edit* I see `parseObject()` exits the run. Lets return the error and exit here in the for-loop to make it obvious whats happeneing?